File: java/Ram.java
class Ram {
    private int capacity;
    private int price;

    public Ram(int capacity, int price) {
        this.capacity = capacity;
        this.price = price;
    }

    public Ram() {
    }

    public int getCapacity() {
        return capacity;
    }

    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public void printRam() {
        System.out.println("Ram Capacity: " + this.capacity + " GB");
        System.out.println("Ram Price: " + this.price);
    }

}

File: php/Ram.php
<?php
class Ram
{
    private $capacity;
    private $price;

    public function __construct($capacity = 0, $price = 0)
    {
        $this->capacity = $capacity;
        $this->price = $price;
    }

    public function getCapacity()
    {
        return $this->capacity;
    }

    public function setCapacity($capacity)
    {
        $this->capacity = $capacity;
    }

    public function getPrice()
    {
        return $this->price;
    }

    public function setPrice($price)
    {
        $this->price = $price;
    }

    public function printRam()
    {
        echo "Ram Capacity: " . $this->capacity . " GB<br/>";
        echo "Ram Price: " . $this->price . "<br/>";
    }
}
